docs/markdown/Python/python/python-linters-and-formatters.md

```diff
@@ -282,3 +282,29 @@ default_section = "THIRDPARTY"
 ```
 
 You may also want to try downgrading to iSort 4.x by setting `version = "isort>=4.6,<5"` in the `[isort]` options scope.
+
+### Black and isort: excluding files
+
+In order to exclude files from being formatted by Black and isort, it may be necessary to explicitly tell those tools to respect skip configuration options. Otherwise, the tools may unconditionally format all files passed to them by the Pants runner.
```

Comment:
```suggestion
In order to exclude files from being formatted by Black and isort, it may be necessary to tell those tools to respect skip configuration options. Otherwise, they may unconditionally format all files passed to them by the Pants runner.
```